modules/Planner/units_addProcess.php

```diff
@@ -125,7 +123,7 @@
                     }
 
                     //ADD BLOCKS
-                    $blockCount = ($_POST['blockCount'] - 1);
+                    $blockCount = ($_POST['blockCount'] - 1) ?? '';
```

Comment:
This is outside the brackets and will not have the desired effect. The null coalesce needs to be on the $_POST variable itself.

Reply:
For the calculation to make sense, I think it should be `0` instead of `''`.